### PR TITLE
Include csv reader and create a unit test for writer.

### PR DESCRIPTION
**Pull request by pdsfilho**
Code for showing outcome of CSV reading with field identification.

### CODE REVIEW

**Comment by akafael** [comment]
Suggestion:
 * [ ] Rename `CSV_write` files to `csvutils`
 * [ ] Add read functions 
```
void readCSV(VehicleData* csvdata, int numEntries, const char* filename);
void readCSVLine(VehicleData data, char* csvLine);
```
 * [ ] Move functions from `car.h` and `car.c` to `csvutils`
 * [ ] Remove car files

File: code/test_csvutils.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    3. data.vehicle.angleBrakePedal,
+    4. data.vehicle.requestCarStart,
+    5. data.battery.voltage, 
+    6. data.battery.current, 
+    7. data.bsg.currentMode, 
+    9. data.bsg.rpm, 
+    10. data.bsg.voltage,
+    11. data.bsg.current
+  */
+  char* expectedCsvLine = "1,2,3,4,5,6,0,8,9,10,11\n";
```
Comment (akafael) [inline code/test_csvutils.c:41]:
```suggestion
  char* expectedCsvLine = "1,2,3,1,5,6,0,8,9,10\n";
```

**Review by akafael — CHANGES_REQUESTED** [review]

File: code/test_csvutils.c [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    7. data.bsg.currentMode, 
+    9. data.bsg.rpm,
+    10. data.bsg.voltage,
+    11. data.bsg.current
+    12. engine.rpg,
+    13. engine.velocity,
+    14. engine.gear
+  */
+  
+  char* expectedCsvLine = "1,2,3,4,5,6,0,8,9,10,11,12,13\n";
```
Comment (akafael) [inline code/test_csvutils.c:47]:
```suggestion
  char* expectedCsvLine = "1,2,3,4,5,6,0,8,9,10,11,12,13";
```

**Review by akafael — CHANGES_REQUESTED** [review]

File: code/test_csvutils.c (hunk 2 above)
Comment (pdsfilho) [reply code/test_csvutils.c]: solved, thanks!

**Review by akafael — APPROVED** [review]

**Pull request merged**